python/tests/test_highlevel.py

```diff
@@ -1272,6 +1272,28 @@ def test_at(self):
                     self.assertEqual(t3.interval, t2.interval)
                     self.assertEqual(t3.parent_dict, t2.parent_dict)
 
+    def test_sequence_iteration(self):
```

Comment:
I'm not seeing much in the way of testing negative indexes.

Reply:
Added a trivial one for the last value. I guess that's enough?